auto-jackson-example/src/main/java/com/artemzin/autojackson/Tweet.java

```diff
@@ -1,45 +1,36 @@
 package com.artemzin.autojackson;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.google.auto.value.AutoValue;
 import org.jetbrains.annotations.NotNull;
 
 @AutoValue
-@JsonDeserialize(builder = Tweet.Builder.class)
-@JsonSerialize(as = Tweet.class)
+@JsonDeserialize(builder = AutoValue_Tweet.Builder.class)
 public abstract class Tweet {
 
   @NotNull
   public static Builder builder() {
-    return Builder.builder();
+    return new AutoValue_Tweet.Builder();
   }
 
   @NotNull
-  @JsonProperty("author")
```

Comment:
In general, I'm not against it, but personally I prefer explicit json property names, so that it's clear which field name is mapped where, it also survives code obfuscation/minification (it's quite popular on Android)

I think we can merge it though

Reply:
I think that in this case it's better not to have them, because it implies that they are actually necessary. Maybe a comment would help?

Reply:
Looking further I realize that the reason the CI failed is that I precisely undid what the last commit did, and relied on all of the default ObjectMapper features that are turned off explicitly in TweetTest. I respect that you want to make this work with minimal ObjectMapper magic, so I guess you shouldn't merge this pr.

Reply:
I think there is value in your PR though, I'm not against showcasing that as a separate class maybe and a README reference?